aiohttp/web_urldispatcher.py

```diff
@@ -148,6 +149,12 @@ def __init__(self, method: str,
         self._expect_handler = expect_handler
         self._resource = resource
 
+    @staticmethod
+    def iscoroutinefunction(handler: Any) -> bool:
```

Comment:
The method should not be a part of public API.
Please make it first-class function which name is started from underscore.

Reply:
I exprted this function to helpers.py and changed it to only happen on Python<3.8